{"name": "Testing login with invalid credentials", "status": "broken", "statusDetails": {"message": "FileNotFoundError: [Errno 2] No such file or directory: 'C:\\\\Users\\\\Eldor\\\\PycharmProjects\\\\appetize/stage_config.json'", "trace": "self = <tests.test_login.TestLogin object at 0x0000018F2F664CD0>\ndriver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"e1b34f972143ddbce33bdec8fec0f463\")>\nenvironment = <ENV.STAGE: (2, 'STAGE', 'stage_config')>\nusername = 'wrong_username', password = 'wrong_pass'\nkey = 'invalid_username_txt'\n\n    @allure.feature('Login Page')\n    @allure.title(\"Testing login with invalid credentials\")\n    @allure.testcase(\"https://accunture.testrail.io/index.php?/cases/view/9999\", 'Test Case Link')\n    @pytest.mark.C9999  # TestRail ID\n    @pytest.mark.smoke\n    @pytest.mark.parametrize(\"environment\", [ENV.PROD, ENV.STAGE])\n    @pytest.mark.parametrize(\"username, password, key\", [\n        (\"wrong_username\", \"SuperSecretPassword!\", \"invalid_username_txt\"),\n        (\"tomsmith\", \"wrong_pass\", \"invalid_password_txt\"),\n        (\"wrong_username\", \"wrong_pass\", \"invalid_username_txt\")\n    \n    ])\n    def test_login_invalid_creds(self, driver, environment, username, password, key):\n>       login_page = LoginPage(env=environment)\n\ntest_login.py:40: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n..\\services\\herokuapp\\pages\\login_page\\__init__.py:18: in __init__\n    self.config = CommonHelper.json_result_reader(file_name=env.config)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nfile_name = 'stage_config'\n\n    @staticmethod\n    def json_result_reader(file_name) -> Dict:\n        \"\"\"\n        Open json by provided name  and return\n        :param file_name: Name of Config file without extension\n        :return: dict\n        \"\"\"\n        import json\n        path = str(CommonHelper.get_project_root())\n>       with open(path + '/' + file_name + '.json') as f:\nE       FileNotFoundError: [Errno 2] No such file or directory: 'C:\\\\Users\\\\Eldor\\\\PycharmProjects\\\\appetize/stage_config.json'\n\n..\\utility\\helpers.py:119: FileNotFoundError"}, "steps": [{"name": "Add failed screenshot", "status": "passed", "attachments": [{"name": "Failed screenshot", "source": "4c426482-5fe3-4b0d-978c-070b13e9ceba-attachment.png", "type": "image/png"}], "start": 1637831246020, "stop": 1637831246661}], "attachments": [{"name": "log", "source": "8b8d239a-ac6c-4a20-bb6d-eb5bca083d17-attachment.txt", "type": "text/plain"}, {"name": "stderr", "source": "50635740-e6da-40b3-9ccf-59fecf287f9f-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "username", "value": "'wrong_username'"}, {"name": "password", "value": "'wrong_pass'"}, {"name": "key", "value": "'invalid_username_txt'"}, {"name": "environment", "value": "<ENV.STAGE: (2, 'STAGE', 'stage_config')>"}], "start": 1637831245972, "stop": 1637831245975, "uuid": "42676457-fed0-45d5-b4bd-514feb693543", "historyId": "3e72ebb61822733384d5d68d85c959f0", "testCaseId": "d3de27ef8e58de518c7ec3f299a7ed3d", "fullName": "tests.test_login.TestLogin#test_login_invalid_creds", "labels": [{"name": "feature", "value": "Login Page"}, {"name": "tag", "value": "smoke"}, {"name": "tag", "value": "C9999"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "LAPTOP-LKUFHBJQ"}, {"name": "thread", "value": "27860-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login"}], "links": [{"type": "test_case", "url": "https://accunture.testrail.io/index.php?/cases/view/9999", "name": "Test Case Link"}]}